packages/universal-components/src/RangeDatePicker/DayNames.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { StyleSheet } from '../PlatformStyleSheet';
+import { Text } from '../Text';
+import RangeDateConfig from './RangeDateConfig';
+
+const dayNames = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
```

Comment:
Maybe there should be a way to pass inn the day names, or set some localization 🤔  Since I want to use this in hotels, I need this to be translatable.

Reply:
I can do it this way (it's not perfect, there is probably no better way how to do it in date-fns):
```
const now = new Date();
const arr = eachDay(startOfWeek(now), endOfWeek(now)
arr.reduce((a, d) => {
  a.push(format(d, 'dd'));
  return a;
});
```
https://github.com/date-fns/date-fns/issues/644

What do you think?

Reply:
Or do you want to have it from `props`?

Reply:
The problem with this is that you can't do `import locale from 'date-fns/esm/locale/{localeName}'` in react-native. It all has to be bundled in advance. 

Seing this post also raises an interesting question, some locales have Sunday as the first day of the week, is this calendar prepared to handle this?

Reply:
1) And what about pass localization to component? Something like:

```
import locale from 'date-fns/esm/locale/cs'
import {RangeDatePicker} from '@kiwicom/universal-components'
...
<RangeDatePicker 
  locale={locale}
/>
```

2) Right now it isn't possible to have Sunday as a first day in week, but I believe it shouldn't be hard to update it. Is it OK if I will do it in next PR?

It's mainly about using different config for this function:
https://date-fns.org/v1.9.0/docs/startOfWeek

Reply:
@tbergq I just added start week support and generated names of the week. So right now you can choose which day should be first in the week.

But localization should be resolved in another PR.

Reply:
1. I think I would prefer that you pass in the name array. Passing the locale, you force the consumer to depend on `date-fns` and we you `Intl` in `mobile` repo for these things. 

2. Ok to do it in a later PR